.eslintrc.js

```diff
@@ -46,19 +46,27 @@ module.exports = {
 		{
 			// This lints the codeblocks marked as `javascript`, `js`, `cjs` or `ejs`, all valid aliases
 			// See:
+			// eslint-disable-next-line inclusive-language/use-inclusive-words
 			//  * https://github.com/highlightjs/highlight.js/blob/master/SUPPORTED_LANGUAGES.md)
 			//  * https://www.npmjs.com/package/eslint-plugin-md#modifying-eslint-setup-for-js-code-inside-md-files
-			files: [ '*.md.js', '*.md.javascript', '*.md.cjs', '*.md.ejs' ],
+			files: [ '*.md.js', '*.md.javascript', '*.md.cjs', '*.md.ejs', '*.md.jsx' ],
 			rules: {
 				// These are ok for examples
 				'import/no-extraneous-dependencies': 'off',
+				'jest/expect-expect': 'off',
+				'jsdoc/require-param-description': 'off',
+				'jsx-a11y/click-events-have-key-events': 'off',
```

Comment:
I'm a bit torn about some of these.

Examples would usually be used from developers to write new code; and if we have examples with a11y issues, then we consciously provoke future code with a11y issues. I'm personally against that.

What do you think?

Reply:
That makes sense.

Do you think I can merge this PR as it is right now (including merging the other 25 sub-PRs to this one, and then merge it in one go to master), and then solve each rule across the whole repo?

Specifically, what rules do you think we should _not_ disable?

Reply:
Yeah, happy to ship it now and fix later 👍  

I was talking specifically about all of those `jsx-a11y` rules.